README.md

```diff
@@ -1,241 +1,296 @@
+the current user to the `video` group and/or `render` group:
 
+```bash
+sudo usermod -a -G video $USER
+sudo usermod -a -G render $USER
 ```
-python -c "import dpctl; dpctl.lsplatform()"
-```
-
-##### Install numba-dpex from source (2/2)
 
-Install and activate the Intel oneAPI DPC++ compiler shipped with the [Intel oneAPI base toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html).
-
-For instance, in Ubuntu, once the apt repo has been configured:
+##### Setup a conda environment for numba-dpex (2/2)
 
+To setup a [conda](https://docs.conda.io/en/latest/) conda environment, run first
+```bash
+export CONDA_DPEX_ENV_NAME=my-dpex-env
+conda create -n $CONDA_DPEX_ENV_NAME numba-dpex intel::dpcpp_linux-64 -c dppy/label/dev -c conda-forge -c intel
 ```
-sudo apt update
-sudo apt install intel-basekit
-source /opt/intel/oneapi/compiler/latest/env/vars.sh
+An additonal command is currently required to work around missing Intel CPU opencl
+runtime activation:
+```bash
+conda env config vars set OCL_ICD_FILENAMES_RESET=1 OCL_ICD_FILENAMES=libintelocl.so -n $CONDA_DPEX_ENV_NAME
 ```
 
-Ensure that the `icx` command can be found in the PATH with the command:
+This will create an environment named `my-dpex-env` (that you can change to your
+liking) containing the package `numba_dpex`, all of its dependencies, and adequate
+configuration.
 
-```
-which icx
-```
-
-Install numba-dpex in the same conda env as previously:
+The corresponding development branch of scikit-learn must be installed from source and
+that requires a separate conda environment. The following sequence of commands will
+create the appropriate conda environment, build the scikit-learn binary, then remove
+the environment:
 
-```
-git clone https://github.com/IntelPython/numba-dpex/
-cd numba-dpex
-pip install -e . --no-build-isolation
+```bash
+conda activate $CONDA_DPEX_ENV_NAME
+export DPEX_PYTHON_VERSION=$(python -c "import platform; print(platform.python_version())")
+export DPEX_NUMPY_VERSION=$(python -c "import numpy; print(numpy.__version__)")
+conda create -n sklearn-dev -c conda-forge "python==$DPEX_PYTHON_VERSION" "numpy==$DPEX_NUMPY_VERSION" scipy cython joblib threadpoolctl pytest compilers
+conda activate sklearn-dev
+git clone https://github.com/ogrisel/scikit-learn -b wip-engines
+cd scikit-learn
+git checkout fdaf97b5b90e18fc63483de9455970123208c9bb
+python setup.py bdist_wheel
+conda activate $CONDA_DPEX_ENV_NAME
+cd dist/
+pip install *.whl
+unset DPEX_PYTHON_VERSION
+unset DPEX_NUMPY_VERSION
+conda deactivate
+conda env remove -n sklearn-dev
+cd ../../
+rm -Rf scikit-learn
 ```
 
-Important: in order to use `numba-dpex`, the `llvm-spirv` compiler is required
-to be in the PATH. This can be achieved with:
-
-```
-$ export PATH=/opt/intel/oneapi/compiler/latest/linux/bin-llvm:$PATH
+Finally, activate the environment with the command:
+```bash
+conda activate my-dpex-env
```

Comment:
```suggestion
conda activate $CONDA_DPEX_ENV_NAME
```

Reply:
I' go against this suggestion, because the environment variable is useful for parameterizing the name of the env in the build instructions where it's used many times, but for using, the user will prefer using the real name